configure.ml

```diff
@@ -12,11 +12,11 @@
 #load "str.cma"
 open Printf
 
-let coq_version = "8.11+alpha"
+let coq_version = "8.12+alpha"
 let coq_macos_version = "8.10.90" (** "[...] should be a string comprised of
```

Comment:
Sorry, I had missed that this number should have been bumped as well:

```suggestion
let coq_macos_version = "8.11.90" (** "[...] should be a string comprised of
```

Reply:
Fixed thanks, I was under the impression we had removed that variable.